### PR TITLE
fix: typo on websocket error handling

### PR DESCRIPTION
**Pull request by joaoopereira**
this PR will fix a typo introduced in my previous PR #1327.

### CODE REVIEW

**Comment by joaoopereira** [comment]
@mcallegari i'm also working on a new webaccess for QLC+, with better responsive support for mobile devices.
As soon as I have a beta version I will share with you. :)

Many thanks for all your work with this application for all this years

File: webaccess/res/websocket.js [hunk 1]
```diff
@@ -29,20 +29,17 @@ function connect() {
...
   setTimeout(function () {
     connect();
   }, 1000);
-};
+ };
 
-websocket.onerror = function (err) {
-  console.error("QLC+ connection encountered error: ", err.message, "Closing socket");
+ websocket.onerror = function(ev) {
+  console.error("QLC+ connection encountered error. Closing socket");
```
Comment (mcallegari) [inline webaccess/res/websocket.js:40]: Codacy has a fix for the issue: [Insert `··`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9368415)
```suggestion
    console.error("QLC+ connection encountered error. Closing socket");
```

File: webaccess/res/websocket.js [hunk 2]
```diff
@@ -29,20 +29,17 @@ function connect() {
...
+  console.log("QLC+ connection is closed. Reconnect will be attempted in 1 second.");
   setTimeout(function () {
     connect();
   }, 1000);
-};
+ };
 
-websocket.onerror = function (err) {
-  console.error("QLC+ connection encountered error: ", err.message, "Closing socket");
+ websocket.onerror = function(ev) {
```
Comment (mcallegari) [inline webaccess/res/websocket.js:39]: Codacy has a fix for the issue: [Replace `websocket.onerror·=·function` with `·websocket.onerror·=·function·`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9368415)
```suggestion
  websocket.onerror = function (ev) {
```

File: webaccess/res/keypad.html [hunk 3]
```diff
@@ -95,19 +95,19 @@
   websocket.onopen = function(ev) {
     isConnected = true;
   };
-  websocket.onclose = function(ev) {
-    console.log(
-      "QLC+ connection is closed. Reconnect will be attempted in 1 second.",
-      e.reason
-    );
+
+  websocket.onclose = function (ev) {
```
Comment (mcallegari) [inline webaccess/res/keypad.html:99]: Seems like ev is not used here and in onerror.
Please either use it or remove it, thanks

File: webaccess/res/simpledesk.js [hunk 4]
```diff
@@ -28,22 +28,22 @@ function getPage(uni, page) {
...
     getPage(1, 1);
    };
-   websocket.onclose = function(ev) {
-    console.log(
-      "QLC+ connection is closed. Reconnect will be attempted in 1 second.",
-      e.reason
-    );
+
+   websocket.onclose = function() {
+    console.log("QLC+ connection is closed. Reconnect will be attempted in 1 second.");
```
Comment (mcallegari) [inline webaccess/res/simpledesk.js:36]: Codacy has a fix for the issue: [Replace `"QLC+·connection·is·closed.·Reconnect·will·be·attempted·in·1·second."` with `⏎······"QLC+·connection·is·closed.·Reconnect·will·be·attempted·in·1·second."⏎····`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9368415)
```suggestion
    console.log(
```

**Pull request merged**